ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/versions/fulu/helpers/MiscHelpersFuluTest.java

```diff
@@ -237,6 +240,34 @@ public void emptyInclusionProofFromRealNetwork_shouldFailValidation() {
         .isFalse();
   }
 
+  @Test
+  public void testCalculateCustodyGroupCount() {
```

Comment:
Refactor to `ParameterizedTest`, see `getExtendedSampleCountReturnsCorrectValues` above for an example. Also I'd input total balance in ETH for readability and multiply it by billion in the body of the test.